Repo.rb

```diff
@@ -81,12 +105,16 @@ def save_to_disk
   def add(files, options)
     files_in_repo, files_in_sub_repos = files
       .group_by { |file| File.dirname(file)}
-      .partition{ |k, v| k == @current_dir}
+      .partition{ |k, v| k == @location}
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.<br>Space missing to the left of {.<br>Space missing inside }.